src/schemas/base.js

```diff
@@ -24,19 +22,11 @@ export const baseNodes = {
 			},
 		],
 		toDOM: (node) => {
-			return ['p', { class: node.attrs.class }, 0];
-		},
-		toStatic: (node, options, isSelected, isEditable, editorProps, children) => {
-			const attrs = {};
-			if (node.attrs && node.attrs.class) {
-				attrs.className = node.attrs.class;
-			}
-			const emptyChildren = <br />;
-			return (
-				<p {...attrs} key={node.currIndex}>
-					{children || emptyChildren}
-				</p>
-			);
+			/* On server rendering, the placeholder <br /> tag is not inserted. */
```

Comment:
What happens on the server side that makes it so the `<br>` is not inserted?

Reply:
I think `<br />` is a contenteditable (or perhaps Prosemirror) thing that is used to keep space in empty paragraphs so your cursor has somewhere to click into. In pure-html land three consecutive, empty `<p></p>` tags would take up zero-height (ignoring margins), but in content-editable land, you expect those to each have `1em` height.